src/components/renderer/form-new-request.vue

```diff
@@ -107,6 +108,15 @@ export default {
           this.processes[category.name].push(process);
         }
       }
+    },
+    fetchData(selectedOption) {
```

Comment:
Start new Requests is not necessary the filters

Reply:
Solved.